src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -749,7 +749,7 @@ MulticopterPositionControl::task_main()
 			matrix::Quatf q_sp = matrix::Eulerf(_att_sp.roll_body, _att_sp.pitch_body, _att_sp.yaw_body);
 			q_sp.copyTo(_att_sp.q_d);
 			_att_sp.q_d_valid = true;
-			_att_sp.thrust = 0.0f;
+			_att_sp.thrust_z = 0.0f;
```

Comment:
General comment: should we ensure that thrust components X and Y are equal to 0 in MC (and that components Y and Z are null in FW)?
```cpp
// MC
_att_sp.thrust_x = 0.0f;
_att_sp.thrust_y = 0.0f;
_att_sp.thrust_z = value;
```

Reply:
@jlecoeur good point! Will do.